CONTRIBUTING.md

```diff
@@ -119,9 +119,9 @@ These documents provide guidance creating a well-crafted commit message:
 
 #### Contributing to Documentation
 
-The [Clarity documentation website](https://vmware.github.io/clarity/documentation) is also housed in this repository under the `new-website` branch.  You can contribute to the documentation by submitting pull requests against that branch.
+The [Clarity documentation website](https://vmware.github.io/clarity/documentation) is also housed in this repository under the `website` branch.  You can contribute to the documentation by submitting pull requests against that branch.
```

Comment:
There is actually a wiki page that talks about how to use the `website` branch, which is more detailed that whats in the README. Its not as simple as it states in the README, so can you update the text 

```
The [Clarity documentation website](https://vmware.github.io/clarity) is also housed in this repository under the `website` branch.  You can contribute to the documentation by submitting pull requests against that branch. Details about how to setup and run the website locally can be found on the wiki at https://github.com/vmware/clarity/wiki/Building-the-Website.   
```